pkg/resource/cluster.go

```diff
@@ -105,6 +144,90 @@ func (cluster Cluster) StatefulSetName() string {
 	return cluster.Name()
 }
 
+func (cluster Cluster) JobName() string {
+	return cluster.Name()
```

Comment:
Change the job name to include "-version-checker"

Reply:
done